cloudinit/distros/bsd.py

```diff
@@ -86,12 +88,15 @@ def package_command(self, command, args=None, pkgs=None):
         if pkgs is None:
             pkgs = []
 
-        # TODO neither freebsd nor netbsd handles a command 'upgrade'
-        # provided by cloudinit/config/cc_package_update_upgrade_install.py
         if command == 'install':
             cmd = self.pkg_cmd_install_prefix
         elif command == 'remove':
             cmd = self.pkg_cmd_remove_prefix
+        elif command == 'update':
+            if not self.pkg_cmd_update_prefix:
+                LOG.debug('Cannot set the locale.')
```

Comment:
this message is wrong.

Reply:
Oops. Thank you. I removed the debug message.